pkg/nsx/services/vpc/builder.go

```diff
@@ -61,9 +65,12 @@ func buildNSXVPC(obj *v1alpha1.VPC, nc common.VPCNetworkConfigInfo, cluster stri
 		vpc = nsxVPC
 	} else {
 		// for creating vpc case, fill in vpc properties based on networkconfig
-		vpcName := util.GenerateDisplayName(obj.Name, obj.GetNamespace(), cluster, "", "")
+		vpcName := sharedVPCName
```

Comment:
although functional work, but the logic order seems a little odd

Reply:
could removed as discussed